internals/daemon/daemon.go

```diff
@@ -717,21 +717,55 @@ func (d *Daemon) doReboot(sigCh chan<- os.Signal, waitTimeout time.Duration) err
 	return fmt.Errorf("expected reboot did not happen")
 }
 
-var shutdownMsg = "reboot scheduled to update the system"
+var (
+	rebootMsg     = "reboot scheduled to update the system"
```

Comment:
Should this be a `const`, or maybe even just a literal string in the one place it's used?

Reply:
Changed the the proposed ```const``` for now for minimum change. I think this could be a separate point of discussion, because I am not sure this single global specific message will be solution going forward. It feels better to supply it with the reboot request so that context specific messages can be supplied. For now, I would prefer to keep the scope of this PR strictly to allow ```shutdown```-less reboots, if that is OK.